rules-tests/Removing/Rector/FuncCall/RemoveFuncCallRector/Fixture/remove_var_dump_in_expressions.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\Removing\Rector\FuncCall\RemoveFuncCallRector\Fixture;
+
+class SomeAction
+{
+    public function run()
+    {
+        if (var_dump('something')) {
```

Comment:
this should not be removed, can be skipped.

Reply:
So if the FuncCall to be removed is part of an expression or statement, it will just be skipped?

Reply:
If not part of Expression, eg If_ cond, it should just be skipped, that will be safe

Reply:
Yes, that will be the safest option. But that means now that we can't use  `getNodeTypes()` with `PhpParser\Node\Stmt\Expression`, as this way we will only be able to filter the functions belonging to other expressions, right? Sorry for making so many questions 😅.

Reply:
You need to define per parent stmt, that need to be carefully added, eg;

```
If_ -> cond
```

You can create separate PR for that, that will be easy to review/merge